prelude/__internal__/TestTools.mad

```diff
@@ -1,25 +1,259 @@
+import HP from "HitPoint"
 import W from "Wish"
 import IO from "IO"
 import S from "String"
 import L from "List"
 import F from "Function"
-
-
-data AssertionError = AssertionError a a
+import FS from "FileSystem"
+import N from "Number"
+import Math from "Math"
+import Maybe from "Maybe"
+import T from "Tuple"
 
 
 colorString :: String -> String -> String
-colorString = (color, v) => (color ++ v ++"\x1b[0m")
+colorString = (color, v) => color ++ v ++"\x1b[0m"
 
 red = colorString("\x1b[31m")
 green = colorString("\x1b[32m")
 
-formatSuccessOutput = (description, result) => (
-  green("\u2713 " ++ description)
+
+data AssertionError = AssertionError a a
+
+
+data LcovLine
```

Comment:
I will move all the Lcov stuff into a separate module that will mostly be used to read / edit / create lcov info files.